### PR TITLE
Cleanup some file/io reader in devcore

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: libdevcore/CommonIO.h [hunk 1]
```diff
@@ -32,7 +32,10 @@ namespace dev
 
 /// Retrieve and returns the contents of the given file as a std::string.
 /// If the file doesn't exist or isn't readable, returns an empty container / bytes.
-std::string contentsString(std::string const& _file);
+std::string readFileAsString(std::string const& _file);
+
+/// Retrieve and returns the contents of standard input (until EOF).
+std::string readStandardInput();
```
Comment (axic) [inline libdevcore/CommonIO.h:38]: Alternatively instead of this we could have `readEntireStream(istream& _stream)` or `readEntireStreamAsString`.

**Comment by axic** [comment]
@chriseth can this be merged?

File: lllc/main.cpp [hunk 2]
```diff
@@ -118,14 +118,7 @@ int main(int argc, char** argv)
 
 	string src;
 	if (infile.empty())
-	{
-		string s;
-		while (!cin.eof())
-		{
-			getline(cin, s);
-			src.append(s);
```
Comment (chriseth) [inline lllc/main.cpp:126]: Oh wow, so lll always discarded the newline?

**Review by chriseth — APPROVED** [review]

File: lllc/main.cpp (hunk 2 above)
Comment (axic) [reply lllc/main.cpp:126]: Yep

**Pull request merged**